<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsultaBienes</class>
 <widget class="QDialog" name="ConsultaBienes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Consulta de Bienes</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Filtro:</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkSector">
        <property name="text">
         <string>Sectores</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkClase">
        <property name="text">
         <string>Clase</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Sector:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="sectores">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grupoEspecificacion">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Especificacion Bien:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Cuenta: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="numeroCuenta"/>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Clase:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="nombreClase">
        <property name="maxLength">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QToolButton" name="btmBuscarClases">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="5">
       <widget class="QLineEdit" name="descripcion">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelResultados">
     <property name="text">
      <string>Resultados: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tablaBienes"/>
   </item>
   <item>
    <widget class="QLabel" name="valorListado">
     <property name="text">
      <string>Valor del Listado: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="seleccionarBien">
        <property name="text">
         <string>Seleccionar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkClase</sender>
   <signal>clicked(bool)</signal>
   <receiver>grupoEspecificacion</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkSector</sender>
   <signal>clicked(bool)</signal>
   <receiver>sectores</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConsultaBienes</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
